python/mxnet/model.py

```diff
@@ -456,6 +456,25 @@ def _check_arguments(self):
         assert(self.symbol is not None)
         self.argument_checked = True
 
+        # skip invalid arg_arams and aux_params
```

Comment:
It is OK, but it is not convenient to automatically separate out the fully connected fine-tune layers, and make their learning rate 10 times larger.

Current API for fine-tuning requires manually hacking on the codes, while this PR makes the whole process automatic.